### PR TITLE
Split the tutorials into two parts

### LINKED ISSUE #234: Improve the sidebar for Get Started
Here are some suggestions for the [入门教程](https://docs.gmt-china.org/latest/tutorial/) sections:
- [x] How about changing the order of [理解图层](https://docs.gmt-china.org/latest/tutorial/layers/) and [绘制直方图](https://docs.gmt-china.org/latest/tutorial/histogram/)? I feel the concept of layering should be the last tutorial.
~~- [ ] How about adding a new section in GMT 入门 (e.g., 绘图模式) and move [GMT单行模式](https://docs.gmt-china.org/latest/tutorial/oneliner/), [GMT图中图模式](https://docs.gmt-china.org/latest/tutorial/inset/), [GMT子图模式](https://docs.gmt-china.org/latest/tutorial/subplot/) and [GMT多图模式](https://docs.gmt-china.org/latest/tutorial/multifig/) there ? I feel new GMT users won't need to learn those four sections until they really have to plot inset and multiple subfigs.~~
~~- [ ] [绘制直方图](https://docs.gmt-china.org/latest/tutorial/histogram/) is great, but I think it's better to put it in another place cause most new GMT users don't need to plot a histogram, right? However, it seems there is no better section to include it. How about adding a new section in GMT 进阶 (e.g., 进阶教程)? #180 is also an advanced tutorial.~~

### PR DESCRIPTION
**Pull request by seisman**
Closes #234.

### CODE REVIEW

**Comment by core-man** [comment]
The sidebar looks strange. Shall we just split it into two files?

**Comment by seisman** [comment]
> The sidebar looks strange.

What do you mean by "strange"? They are two chapters but share the same index file. In this way, we don't have to put them into separate directories.

File: source/tutorial/index.rst [hunk 1]
```diff
@@ -1,6 +1,10 @@
 入门教程
 ########
 
+《入门教程》通过一系列简单的示例介绍了如何使用 GMT 实现基本的绘图功能，
+如绘制底图、海岸线、线段、符号、文字等。初学者应仔细阅读《入门教程》，
+并掌握 GMT 的基本用法。
```
Comment (core-man) [inline source/tutorial/index.rst:6]:
```suggestion
《入门教程》通过一系列简单的示例介绍如何使用 GMT 实现最基本的绘图功能，
如绘制底图、海岸线、线段、符号、文字等。初学者应仔细阅读《入门教程》，
并掌握 GMT 的基本用法。
```

File: source/tutorial/index.rst [hunk 2]
```diff
@@ -14,8 +18,17 @@
    texts
    legend
    earth-relief
-   histogram/index
    layers
+
+进阶教程
+########
+
+《进阶教程》展示了如果使用 GMT 实现更复杂的绘图功能。
```
Comment (core-man) [inline source/tutorial/index.rst:26]:
```suggestion
《进阶教程》展示如何使用 GMT 实现更复杂的绘图功能。
```

**Comment by core-man** [comment]
> > The sidebar looks strange.
> 
> What do you mean by "strange"? They are two chapters but share the same index file. In this way, we don't have to put them into separate directories.

If the index is long, I feel new GMT users may be confused at the begining. How about using `hlist` to make 3 columns if we want to keep a single index page?

**Comment by seisman** [comment]
> If the index is long, I feel new GMT users may be confused at the begining. How about using `hlist` to make 3 columns if we want to keep a single index page?

The index is not that long now. BTW, using `hlist` may not a good idea, because it means more maintaining burden on us. When we add a new tutorial, we have to add the new tutorial in the list and the toctree. It also doesn't work well in PDF.

**Review by core-man — APPROVED** [review]

**Comment by core-man** [comment]
> > If the index is long, I feel new GMT users may be confused at the begining. How about using `hlist` to make 3 columns if we want to keep a single index page?
> 
> The index is not that long now. BTW, using `hlist` may not a good idea, because it means more maintaining burden on us. When we add a new tutorial, we have to add the new tutorial in the list and the toctree. It also doesn't work well in PDF.

Okay. One-page is still okay now.

**Pull request merged**